### DISCUSSION TITLE
throttle

### DISCUSSION THREAD

**Discussion by Revil2020** [flybywiresim/aircraft#9250] [Help]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-10-31T15:54:41+00:00",
    "ref": "refs/tags/v0.12.0",
    "sha": "2fe89113291f2478347dde0dc3228b55dc924f74",
    "actor": "FoxtrotSierra6829",
    "event_name": "manual",
    "pretty_release_name": "stable/v0.12.0",
    "version": "a380x-v0.12.0-rel.2fe8911"
}
```

### Describe the bug

I don't have neutral with my joystick: when I put it in neutral, the revs increase to 70%.

### Expected behavior

The engine speed is idling.

### Steps to reproduce

nul

### Discord Username (optional)

Revil2000

**Comment by alepouna** [comment]
Hey, this seems to be a technical issue with the configuration of the aircraft rather than a bug. We recommend you try to reach out via our Discord Server first (https://discord.gg/flybywire) and reach out to our support channels! It will allow for easier communication and hopefully resolve the issue. 

*This issue will therefore be closed and be transferred to a discussion, for if you are unable to join our Discord community.*
